types/generate-ctrl-c-event/index.d.ts

```diff
@@ -0,0 +1,17 @@
+// Type definitions for generate-ctrl-c-event 1.2
```

Comment:
```suggestion
// Type definitions for generate-ctrl-c-event 2.0
```

@kazarmy If you don't object, I will make a major release of `generate-ctrl-c-event`, removing the type definitions and bumping to version 2.0. 

> If you do so, could you just do a point release? Otherwise I'd have to update the typings version on DefinitelyTyped.
> - https://github.com/zenflow/generate-ctrl-c-event/issues/9#issuecomment-812492106

Going "by the book" this is a semver-major change since it is potentially breaking and users may need to install the `@types/generate-ctrl-c-event` package for things to work as before, if they were using the built-in TS types.

It shouldn't be a big problem to update this PR, since you can just click to apply this suggested change, right?

Reply:
I think this is a good idea. I didn’t realize that the typings were going to be _removed_ from the implementation library, but that makes me feel better about this PR in general. I was hesitant primarily due to the duplication.

Reply:
@andamanastireanu I really appreciate your concern and help here! Thanks!

Reply:
I opened this PR to remove the typings from the implementation library: https://github.com/zenflow/generate-ctrl-c-event/pull/10

@andamanastireanu I don't think there is anything else to do to prepare the implementation library, correct? 😃

Reply:
> It shouldn't be a big problem to update this PR, since you can just click to apply this suggested change, right?

DefinitelyTyped is very strict and won't allow me to refer to an npm version that doesn't exist (https://github.com/DefinitelyTyped/DefinitelyTyped/runs/2258157522#step:5:80) so I've reverted the version change. So I think it's either the package goes first or there's going to be another DefinitelyTyped PR.

Reply:
@kazarmy I just merged my PR and v2 was released: https://github.com/zenflow/generate-ctrl-c-event/releases/tag/v2.0.0
So now you can apply the suggested change and CI should pass and then this is ready to merge I think.

```suggestion
// Type definitions for generate-ctrl-c-event 2.0
```